core-bundle/src/InsertTag/Flag/PhpFunctionFlag.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\InsertTag\Flag;
+
+use Contao\CoreBundle\DependencyInjection\Attribute\AsInsertTagFlag;
+use Contao\CoreBundle\InsertTag\InsertTagFlag;
+use Contao\CoreBundle\InsertTag\InsertTagResult;
+
+#[AsInsertTagFlag('addslashes')]
+#[AsInsertTagFlag('strtolower')]
+#[AsInsertTagFlag('strtoupper')]
+#[AsInsertTagFlag('ucfirst')]
+#[AsInsertTagFlag('lcfirst')]
+#[AsInsertTagFlag('ucwords')]
+#[AsInsertTagFlag('trim')]
+#[AsInsertTagFlag('rtrim')]
+#[AsInsertTagFlag('ltrim')]
+#[AsInsertTagFlag('urlencode')]
+#[AsInsertTagFlag('rawurlencode')]
+class PhpFunctionFlag implements InsertTagFlagInterface
+{
+    public function __invoke(InsertTagFlag $flag, InsertTagResult $result): InsertTagResult
+    {
+        if (
```

Comment:
Just FYI, we could use reflection an check the attributes here. I guess performance wise it would be pretty much irrelevant and it's one mistake less one could make in this class. But we can also leave as is :D